components/add_groups_to_channel_modal/add_groups_to_channel_modal.jsx

```diff
@@ -13,7 +13,7 @@ import {localizeMessage} from 'utils/utils.jsx';
 
 import MultiSelect from 'components/multiselect/multiselect.jsx';
 import groupsAvatar from 'images/groups-avatar.png';
-import AddIcon from 'components/icon/add_icon';
+import AddIcon from 'components/widgets/icons/fa_add_icon';
```

Comment:
it is not `add_icon`?

Reply:
Nope, because we had `components/icon/xxxx_icon` for font awesome based icons and `components/svg/xxxx_icon` for svg based icons, and some of them had the same name, I migrated them using the `fa_` prefix for the font awesome icons.